demo/scripts/multi_planner.py

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from moveit.task_constructor import core, stages
+from moveit_commander.roscpp_initializer import roscpp_initialize
+import time
+
+roscpp_initialize("mtc_tutorial_multi_planner")
+
+ompl_pipelinePlanner = core.PipelinePlanner("ompl")
+ompl_pipelinePlanner.planner = "RRTConnectkConfigDefault"
+pilz_pipelinePlanner = core.PipelinePlanner("pilz_industrial_motion_planner")
+pilz_pipelinePlanner.planner = "PTP"
+multiPlanner = core.MultiPlanner()
+multiPlanner.add(ompl_pipelinePlanner)
+multiPlanner.add(pilz_pipelinePlanner)
```

Comment:
Probably you want to switch the order of these planners: the PTP planner that is prone to fail should come first.